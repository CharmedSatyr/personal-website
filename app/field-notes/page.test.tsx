import FieldNotes from "@/app/field-notes/page";
import { render } from "@testing-library/react";

describe("Field Notes", () => {
	it("should not deviate unexpectedly from the last snapshot", () => {
		const { container } = render(<FieldNotes />);

		expect(container).toMatchSnapshot();
	});
});
