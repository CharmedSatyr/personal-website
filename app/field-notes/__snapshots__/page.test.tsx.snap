// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Field Notes should not deviate unexpectedly from the last snapshot 1`] = `
<div>
  <h1
    class="title"
  >
    Field Notes
  </h1>
  <section>
    <a
      class="mb-4 flex items-end justify-start gap-4"
      href="/field-notes/how-i-learned-to-code"
    >
      <time>
        2024-06-25
      </time>
      <h3
        class="hyperlink"
      >
        How I Learned to Code
      </h3>
    </a>
  </section>
</div>
`;
